src/components/menu/menu.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { FC, ReactElement } from 'react';
+import { Link } from 'react-router-dom';
+import cn from 'classnames';
+
+import { GAME_NAME } from 'constants/commonConstants';
+import { MENU_ITEMS } from 'constants/menuConstants';
+
+import './menu.scss';
+
+export type TMenuItem = {
+  title: string;
+  route?: string;
+  action?: string;
+}
+
+interface IMenu {
+  menuItems?: TMenuItem[];
+  isShown?: boolean;
+  handleAction?: (action: string) => void;
+}
+
+const Menu: FC<IMenu> = ({ menuItems = MENU_ITEMS, isShown = true, handleAction }): ReactElement => {
+  return (
+    <div className={cn('menu', (isShown ? '' : ' hidden'))}>
```

Comment:
❗️
А вот тут уже можно и через `&&` написать
```suggestion
    <div className={cn('menu', isShown && ' hidden')}>
```